src/foam/u2/view/FObjectView.js

```diff
@@ -89,9 +87,13 @@ foam.CLASS({
       value: true
     },
     {
-      class: 'Boolean',
-      name: 'persistData',
-      documentation: 'Allows data to persist to selected class instances.'
+      class: 'Object',
```

Comment:
do you have an example of how this will be used/set?

Reply:
Sure, I'll include the use of a predicate as well. Here's an example.:
```javascript
view: function(_, X) {
  let predicate = expr.AND(
      expr.EQ(foam.strategy.StrategyReference.DESIRED_MODEL_ID, 'foam.nanos.auth.User'),
      expr.IN(foam.strategy.StrategyReference.STRATEGY, [foam.lookup('foam.nanos.auth.SomeUserClass'), foam.lookup('foam.nanos.auth.AnotherUserClass') ])
  );
  return foam.u2.view.FObjectView.create({
    data: myData,
    of: foam.nanos.auth.User,
    persistantData: { website: X.data.user.website },
    predicate: predicate
 }, X);
```
- The FObjectView will persist X.data.user.website throughout any class selections.
- Strategizer class options will include SomeUserClass and AnotherUserClass excluding all other user subclasses. Granted the user has permissions to the strategies and if those classes exist.

Reply:
Spelling: persistent

Reply:
I still don't get it from your explanation. Can you elaborate?

Reply:
@kgrgreer 
`persistentData` will be copied over to any objects created after a strategizer class selection.
`predicate` can omit or include additional strategizer class options.